### PR TITLE
Ask for confirmation when following an organisation invite link

### PR DESCRIPTION
**Pull request by reinhardt**
Note that this removes the permission check on the organisation logo view because the design implies that a user should see the logo even if they're not yet a member of the organisation.

syslabcom/scrum#594

### CODE REVIEW

**Comment by reinhardt** [comment]
FWIW `test_confirmation_link` passes in my local installation

File: src/euphorie/client/browser/templates/confirm-organisation-invite.pt [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xmlns:meta="http://xml.zope.org/namespaces/meta"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      meta:interpolation="true"
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/confirm-organisation-invite.pt:7]:
```suggestion
```

File: src/euphorie/client/browser/templates/confirm-organisation-invite.pt [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xmlns:meta="http://xml.zope.org/namespaces/meta"
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/confirm-organisation-invite.pt:4]:
```suggestion
```

File: src/euphorie/client/browser/organisation.py [hunk 3]
```diff
@@ -336,6 +341,29 @@ def organisation(self):
...
+        return user
+
+    @property
+    def inviter_fullname(self):
+        user = self.inviter
+        if user is None:
+            return ""
+        if not (user.last_name or user.first_name):
+            return user.loginname
+        return " ".join(filter(None, (user.first_name, user.last_name)))
```
Comment (ale-rt) [inline src/euphorie/client/browser/organisation.py:365]:
```suggestion
        return user.title
```
See: https://github.com/euphorie/Euphorie/blob/6aad346174f2a57a1bc2e9bc340862658bc62b77/src/euphorie/client/model.py#L516-L526

**Review by ale-rt — APPROVED** [review]
Minor things left

**Pull request merged**